routes/utils/errors.js

```diff
@@ -16,6 +16,12 @@ module.exports.INVALID_SESSION = hoodieError({
   status: 401
 })
 
+module.exports.USER_NOT_FOUND = hoodieError({
+  name: 'Unauthorized',
+  message: 'Invalid credentials',
```

Comment:
According to the [spec](http://docs.accountjsonapi.apiary.io/#reference/current-user/session/sign-in) the error message should be "Invalid credentials", let me know if you prefer to return something else instead